check_patch.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+pip install pre-commit
```

Comment:
You could just put this into the requirements and run the `pre-commit run --all-files` directly.

Reply:
Also, putting this file in the root dir is ugly :slightly_smiling_face:

Reply:
Agree, though I thought that creating `scripts` dir will be premature for single scirpt